test/integrations/destinations/pardot/dataDelivery/business.ts

```diff
@@ -0,0 +1,393 @@
+                  zip: null,
+                  phone: null,
+                  fax: null,
+                  source: null,
+                  annual_revenue: null,
+                  employees: null,
+                  industry: null,
+                  years_in_business: null,
+                  comments: null,
+                  notes: null,
+                  score: 14,
+                  grade: null,
+                  last_activity_at: null,
+                  recent_interaction: 'Never active.',
+                  crm_lead_fid: null,
+                  crm_contact_fid: null,
+                  crm_owner_fid: '00G2v000004WYXaEAO',
+                  crm_account_fid: null,
+                  salesforce_fid: null,
+                  crm_last_sync: null,
+                  crm_url: null,
+                  is_do_not_email: null,
+                  is_do_not_call: null,
+                  opted_out: null,
+                  is_reviewed: 1,
+                  is_starred: null,
+                  created_at: '2022-01-21 18:21:46',
+                  updated_at: '2022-01-21 18:48:41',
+                  campaign: { id: 42113, name: 'Test', crm_fid: '7012y000000MNOCLL4' },
+                  assigned_to: {
+                    user: {
+                      id: 38443703,
+                      email: 'test_rudderstack@testcompany.com',
+                      first_name: 'Rudderstack',
+                      last_name: 'User',
+                      job_title: null,
+                      role: 'Administrator',
+                      account: 489853,
+                      created_at: '2021-02-26 06:25:17',
+                      updated_at: '2021-02-26 06:25:17',
+                    },
+                  },
+                  Are_you_shipping_large_fragile_or_bulky_items: false,
+                  Calendly: false,
+                  Country_Code: 'AU',
+                  Currency: 'AUD',
+                  Inventory_or_Warehouse_Management_System: false,
+                  Lead_Status: 'New',
+                  Marketing_Stage: 'SAL',
+                  Record_Type_ID: 'TestCompany Lead',
+                  profile: {
+                    id: 304,
+                    name: 'Default',
+                    profile_criteria: [
+                      { id: 1500, name: 'Shipping Volume', matches: 'Unknown' },
+                      { id: 1502, name: 'Industry', matches: 'Unknown' },
+                      { id: 1506, name: 'Job Title', matches: 'Unknown' },
+                      { id: 1508, name: 'Department', matches: 'Unknown' },
+                    ],
+                  },
+                  visitors: null,
+                  visitor_activities: null,
+                  lists: null,
+                },
+              },
+              status: 201,
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'pardot_v0_bussiness_scenario_2',
+    name: 'pardot',
+    description: '[Proxy v0 API] :: pardot fid type upsert',
```

Comment:
The descriptions need to be more elaborative what exactly we are testing and what is the expectations?

Reply:
here we are test email and fid upsert types call,